src/components/vedtak/VedtakInfopanel.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Innholdstittel } from 'nav-frontend-typografi';
+import { restdatoTildato } from '../../utils/datoUtils';
+import { VedtakDTO } from '../../reducers/vedtak';
+import Panel from 'nav-frontend-paneler';
+import VedtakOppsummering from './VedtakOppsummering';
+import VedtakUtbetaltePerioder from './VedtakUtbetaltePerioder';
+import styled from 'styled-components';
+import { Row } from 'nav-frontend-grid';
+
+
+interface IVedtakInfopanel {
```

Comment:
Står I-en i IVedtakInfopanel for interface? Er det konvensjonen i javascript?

Reply:
Yes!